helix-term/src/ui/picker.rs

```diff
@@ -34,6 +34,11 @@ pub const MIN_AREA_WIDTH_FOR_PREVIEW: u16 = 72;
 /// Biggest file size to preview in bytes
 pub const MAX_FILE_SIZE_FOR_PREVIEW: u64 = 10 * 1024 * 1024;
 
+pub trait PickStepper {
+    fn next(&mut self, cx: &mut Context);
+    fn prev(&mut self, cx: &mut Context);
+}
```

Comment:
Can we do this without introducing a new trait for it? A trait feels like the wrong tool.

Maybe we can somehow downcast the `last_picker` Component to a `Picker` and add next/prev there?